playbooks/tasks/portal-role-task.yml

```diff
@@ -34,8 +33,8 @@
 # Setup MongoDB
 # NOTE: MongoDB docker service must be setup before Sia docker service, because
 # skyd depends on mongo.
-- name: Include ensuring MongoDB node is a member of replicaset
-  include_tasks: tasks/portal-role-task-mongo-replicaset-ensure-member.yml
+# - name: Include ensuring MongoDB node is a member of replicaset
+#   include_tasks: tasks/portal-role-task-mongo-replicaset-ensure-member.yml
```

Comment:
I had to comment this out for the migration work. It kept failing. Should be able to add it back in.